### ISSUE TITLE
dependabot not detecting dependency updates (or taking quite some time for it)

### ISSUE THREAD

**Issue by moving-bits** [cgeo/cgeo#13193]
### Tell us your idea!

Discovered by chance that dependabot does not detect all lib updates or takes quite some time for it:
- At the very same time I was pushing a manual PR for updating the VTM dependency, dependabot opened a new PR regarding that VTM update, which had been released about a week ago (whereas it had detected the mapsforge update of the same base project nearly immediately)
- ok.http update is not detected yet, although nearly two weeks old

(There may be others, haven't checked)

Is there anything we can do about this, or do we simply have to live with it?

**Comment by bekuno** [comment]
We have set a limit of 20 - see https://github.com/cgeo/cgeo/blob/master/.github/dependabot.yml Line 8

**Comment by moving-bits** [comment]
> We have set a limit of 20 - see https://github.com/cgeo/cgeo/blob/master/.github/dependabot.yml Line 8

ah, good to know, thanks for the pointer. That probably explains both issues.

I propose to raise this limit a bit, maybe to 25 or even 30, as cross-checking the dependency PR may take some time, some may even be temporarily blocked, and this will prevent other, potentially more urgent dependency PR from being created.

**Comment by fm-sys** [comment]
> I propose to raise this limit a bit, maybe to 25 or even 30, as cross-checking the dependency PR may take some time

On the one hand I understand why you want to increase the limit, on the other hand the PR section is already today way too crowded. Wouldn't it be better if we just merge dependency updates more offensive?

**Comment by moving-bits** [comment]
Yes, the list is currently too long - but that depends on available time for cross-checking.
I guess allowing more dependency PR (to not miss an important one) goes well with more optimistic / progressive merging of those PR.

So in addition to above a second proposal: Shall we merge all currently open material.io-related PR to `master` at the end of today? We are still at the beginning of a release cycle, so we have plenty of time to check for any unwanted side-effects.
Will do so this evening, if no-one stops me...

**Comment by bekuno** [comment]
> I guess allowing more dependency PR (to not miss an important one) goes well with more optimistic / progressive merging of those PR.

:-)
